### DISCUSSION TITLE
How to generate the leaf certificate alone using existing root CA certificate

### DISCUSSION THREAD

**Discussion by BalajiVM** [rabbitmq/tls-gen#47] [General]
Hi Team,

I have a requirement to create custom root CA certificate with 20 years validity and leaf certificate with 2 years validity 
After 2 years, i have to generate a leaf certificate with the same root certificate.
Please let me know how we can generate leaf certificate with same root CA

**Comment by michaelklishin** [comment]
There are sections named "Generate additional … certificate" in the README of the profile(s) that support this feature as of #37.

FWIW tls-gen is meant to generate development certificates, we never meant for these chains to be used for years with the need to renew.
